src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -2841,6 +2841,45 @@ exprt c_typecheck_baset::do_special_functions(
     overflow.type() = bool_typet{};
     return overflow;
   }
+  else if(identifier == CPROVER_PREFIX "enum_is_in_range")
+  {
+    // Check correct number of arguments
+    if(expr.arguments().size() != 1)
+    {
+      error().source_location = expr.source_location();
+      error() << identifier << " expects one argument, "
+              << expr.arguments().size() << " were provided" << eom;
+      throw 0;
```

Comment:
Is it worth adding a test also covering this error condition? You've got a bunch of tests already, adding one more would help tick another box :-)

Reply:
Done